<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidgetSimpleTab</class>
 <widget class="QWidget" name="FilterWidgetSimpleTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="classesLabel">
       <property name="text">
        <string>Classes:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="SelectClassesWidget" name="select_classes" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectClassesWidget</class>
   <extends>QWidget</extends>
   <header>filter_widget/select_classes_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
